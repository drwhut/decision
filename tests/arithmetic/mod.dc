Start~#1

> Mod 2 positive integers
Mod(1651, 7)~#2
Print(#1, #2)~#3

> Mod a negative integer from a positive one
Mod(12, -3)~#4
Print(#3, #4)~#5

> Mod a positive integer from a negative one
Mod(-100, 18)~#6
Print(#5, #6)~#7

> Mod 2 negative integers
Mod(-713, -9)~#8
Print(#7, #8)~#9