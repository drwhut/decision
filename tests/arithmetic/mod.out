6
0
-10
-2
